waiter/src/waiter/scheduler/kubernetes.clj

```diff
@@ -923,7 +933,29 @@
            :resources {:limits {:memory memory}
                        :requests {:cpu cpu :memory memory}}
            :volumeMounts [{:mountPath "/srv/www"
-                           :name "user-home"}]})))))
+                           :name "user-home"}]}))
+
+      ;; Optional envoy sidecar container
+      has-reverse-proxy?
+      (update-in
```

Comment:
Nitpick: this update could be done by a configurable function like we configure the `predicate-fn`. We can do that in a separate PR.

Reply:
Done